checker.js

```diff
@@ -0,0 +1,175 @@
+var fs         = require('fs')
+var Table      = require('cli-table');
+var colors     = require('colors');
+var path       = require('path');
+var Superagent = require('superagent');
+var readline   = require('readline-sync');
+
+var LIST_MODULES_URL = 'https://webtask.it.auth0.com/api/run/wt-tehsis-gmail_com-1?key=eyJhbGciOiJIUzI1NiIsImtpZCI6IjIifQ.eyJqdGkiOiJmZGZiOWU2MjQ0YjQ0YWYyYjc2YzAwNGU1NjgwOGIxNCIsImlhdCI6MTQzMDMyNjc4MiwiY2EiOlsiZDQ3ZDNiMzRkMmI3NGEwZDljYzgwOTg3OGQ3MWQ4Y2QiXSwiZGQiOjAsInVybCI6Imh0dHA6Ly90ZWhzaXMuZ2l0aHViLmlvL3dlYnRhc2tpby1jYW5pcmVxdWlyZS90YXNrcy9saXN0X21vZHVsZXMuanMiLCJ0ZW4iOiIvXnd0LXRlaHNpcy1nbWFpbF9jb20tWzAtMV0kLyJ9.MJqAB9mgs57tQTWtRuZRj6NCbzXxZcXCASYGISk3Q6c';
+var pkg = path.join(process.cwd(), 'package.json');
+
+module.exports.check = check;
+
+module.exports.sync  = sync;
+
+function readVersion (ver) {
+    return Number.isNaN(Number.parseInt(ver.charAt(0))) ? ver.substring(1, ver.length) : ver;
+}
+
+function checkVersion(local, webtask) {
+    var loc  = local.split('.');
+    var wt   = webtask.split('.');
+    var flag = false;
+
+    for(var i=0; i<loc.length; i++) {
+        if (loc[i] > wt[i]) {
+            flag = true;
+            break;
+        }
+    }
+
+    if (!flag) {
+        for(var i=0; i<loc.length; i++) {
+            if (wt[i] > loc[i]) {
+                flag = true;
+                break;
+            }
+        }
+    }
+
+    return flag;
+}
+
+function findByName (key) {
+    return function (el) {
+        return el.name === key;
+    }
+}
+
+function findByNameAndVersion (key, localVersion) {
+    return function (el) {
+        return el.name === key &&  el.version === localVersion;
+    }
+}
+
+function format(str, length) {
+    var strLength = str.length;
+
+    for(var i = 0; i < (length-strLength); i++) {
+        str = str + ' ';
+    }
+
+    return str;
+}
+
+function sync(opt) {
+    try {
+        fs.statSync(pkg);
+    }
+    catch (err) {
+        return console.log('\n-> "package.json" does not exists\n'.red);
+    }
+
+    try {
+        console.log('\n-> Copying "package.json" to "package.json.orig"\n');
+        fs.createReadStream(pkg).pipe(fs.createWriteStream(path.join(process.cwd(), 'package.json.orig')));
+    }
+    catch (err) {
+        return console.log('\n-> Error trying to copy "package.json"\n'.red);
+    }
+
+    Superagent.get(LIST_MODULES_URL)
```

Comment:
I have a method in `sandboxjs` that I exported called `issueRequest` that takes a `superagent` request (that hasn't had `end` called) and returns a promise. This is useful to deal with the boilerplate of handling `http` vs. `transport` errors: https://github.com/auth0/sandboxjs/blob/master/lib/issueRequest.js